cg/models/scout/scout_load_config.py

```diff
@@ -24,10 +24,10 @@ class ScoutIndividual(BaseModel):
     confirmed_sex: Optional[bool] = None
     father: Optional[str] = None
     mother: Optional[str] = None
-    phenotype: Literal["affected", "unaffected", "unknown"] = None
+    phenotype: PhenotypeStatus = PhenotypeStatus.MISSING
     sample_id: str = None
     sample_name: Optional[str] = None
-    sex: Literal["male", "female", "unknown"] = None
+    sex: Literal[Gender.MALE, Gender.FEMALE, Gender.UNKNOWN] = None
```

Comment:
You put default for phenotype that it is missing if it is missing. But gender you have "none", more consistent if also setting gender to "missing"?

Reply:
PhenotypeStatus.MISSING is also as well, if you check in cg/constants/subject.py

Reply:
Added Gender.missing = None for consistency